client/src/views/Mapping/components/FhirMappingPanel/components/ResourceSelector/Drawer/index.tsx

```diff
@@ -0,0 +1,265 @@
+import { useApolloClient } from "@apollo/react-hooks";
+import {
+  InputGroup,
+  FormGroup,
+  Button,
+  Classes,
+  Drawer as BPDrawer,
+  ControlGroup,
+  Position
+} from "@blueprintjs/core";
+import React from "react";
+import { Mutation } from "react-apollo";
+import { useSelector, useDispatch, useStore } from "react-redux";
+import useReactRouter from "use-react-router";
+
+import { IReduxStore } from "src/types";
+
+import ColumnPicker from "src/views/Mapping/components/ColumnPicker";
+
+import "./style.less";
+
+import {
+  updateFhirResource,
+  deselectFhirResource
+} from "src/services/selectedNode/actions";
+
+import { deleteLocationParams } from "src/services/urlState";
+
+const resourceInfo = require("src/graphql/queries/resourceInfo.graphql");
+const deleteResourceMutation = require("src/graphql/mutations/deleteResource.graphql");
+const updateResource = require("src/graphql/mutations/updateResource.graphql");
+
+interface IProps {
+  title: string;
+  isOpen: boolean;
+  deleteResourceCallback?: () => void;
+  onCloseCallback?: () => void;
+}
+
+const Drawer = ({
+  title,
+  isOpen,
+  deleteResourceCallback,
+  onCloseCallback
+}: IProps) => {
+  const client = useApolloClient();
+  const dispatch = useDispatch();
+  const selectedNode = useSelector((state: IReduxStore) => state.selectedNode);
+  const store = useStore();
+  const toaster = useSelector((state: IReduxStore) => state.toaster);
+  const { history, location } = useReactRouter();
+
+  const [schema, setSchema] = React.useState({});
+  const [label, setLabel] = React.useState("");
+  const [pkOwner, setPkOwner] = React.useState("");
+  const [pkTable, setPkTable] = React.useState("");
+  const [pkColumn, setPkColumn] = React.useState("");
+
+  const onFormSubmit = (updateResource: any) => {
+    return (e: React.FormEvent<HTMLElement>) => {
+      e.preventDefault();
+
+      updateResource({
+        variables: {
+          where: {
+            id: selectedNode.resource.id
+          },
+          data: {
+            label,
+            primaryKeyOwner: pkOwner,
+            primaryKeyTable: pkTable,
+            primaryKeyColumn: pkColumn
+          }
+        }
+      });
+    };
+  };
+
+  const onUpdateCompleted = () => {
+    toaster.show({
+      message: `Successfully updated ${
+        selectedNode.resource.fhirType
+      } properties`,
+      intent: "success",
+      icon: "properties"
+    });
+
+    dispatch(
+      updateFhirResource(
+        selectedNode.resource.id,
+        selectedNode.resource.fhirType,
+        label
+      )
+    );
+  };
+
+  const onUpdateError = (error: any) => {
+    toaster.show({
+      message: "An error occurred while updating properties",
+      intent: "danger",
+      icon: "properties"
+    });
+  };
+
+  const onDeletionCompleted = (data: any) => {
+    toaster.show({
+      icon: "layout-hierarchy",
+      intent: "success",
+      message: `Ressource ${data.deleteResource.fhirType} supprimée pour ${
```

Comment:
you should decide whether to use french or english and stick to it ;)

Reply:
English it is!